src/geom/SkyWcs.cc

```diff
@@ -220,7 +219,9 @@ std::shared_ptr<daf::base::PropertyList> SkyWcs::getFitsMetadata(bool precise) c
             throw LSST_EXCEPT(lsst::pex::exceptions::RuntimeError,
                               "Could not represent this SkyWcs using FITS-WCS metadata");
         } else {
-            // A large FitsTol was not sufficient; write a local TAN WCS approximation
+            // A large FitsTol was not sufficient; get a local TAN WCS approximation
```

Comment:
I don't understand this comment here.  What is FitsTol?

Reply:
Good catch. The comment was out of date. For the record: FitsTol is something AST uses to decide if it can write a WCS out as FITS header cards; code a few lines above sets it very small and that's the only time it's ever set. At one time the code below this comment tried setting it larger to see if AST could write a useful approximation, but I decided the current code was simpler and produced better results.